WrightTools/kit.py

```diff
@@ -637,7 +644,7 @@ def read_headers(filepath):
 
 
 def write_h5(filepath, dictionary):
-    """ Save a python dictionary into an `HDF5 <https://www.hdfgroup.org/HDF5/doc/H5.intro.html>`_
+    """ Save a python dictionary into an `HDF5`_
```

Comment:
It does, and it throws errors if you try to put the same tag in but redefine it...

I don't like not having the link in the docstring itself, but I'm not sure how to resolve that at this time